addons/event/models/event_ticket.py

```diff
@@ -61,11 +62,17 @@ def default_get(self, fields):
         ondelete='cascade', required=True)
     company_id = fields.Many2one('res.company', related='event_id.company_id')
     # sale
-    start_sale_date = fields.Date(string="Registration Start")
-    end_sale_date = fields.Date(string="Registration End")
+    start_sale_date = fields.Datetime(string="Registration Start")
```

Comment:
Same here: would rename fields (_date indicates it contains a date, therefore _datetime should be the new suffix).